stdlib/http/src/main/java/org/ballerinalang/net/http/HttpDispatcher.java

```diff
@@ -255,6 +251,50 @@ private static BValue populateAndGetEntityBody(BMap<String, BValue> inRequest,
         return null;
     }
 
+    /**
+     * Convert a json to the relevant record type.
+     *
+     * @param entityBodyType Represents entity body type
+     * @param bjson          Represents the json value that needs to be converted
+     * @return the relevant ballerina record or object
+     */
+    private static BValue getRecord(BType entityBodyType, BValue bjson) {
+        try {
+            return JSONUtils.convertJSONToStruct(bjson, (BStructureType) entityBodyType);
+        } catch (NullPointerException ex) {
```

Comment:
Again, not a change introduced by this PR, but shouldn't we handle other possible exceptions too? Not sure why we've handled (only) a NPE here?

Reply:
Other exceptions are handled by the calling function.